File: app/src/main/java/com/rr/rhythms/Entities/Point.java
package com.rr.rhythms.Entities;

import java.io.Serializable;

/**
 * Created by Huruk on 7/5/2016.
 */
public class Point implements Serializable {
    private float X;
    private float Y;

    public Point(float x, float y) {
        X = x;
        Y = y;
    }

    public float GetX() {
        return X;
    }

    public float GetY() {
        return Y;
    }
}

File: app/src/main/java/com/rr/rhythms/Entities/Point.kt
package com.rr.rhythms.Entities

import java.io.Serializable

/**
 * Created by Huruk on 7/5/2016.
 */
class Point(private val X: Float, private val Y: Float) : Serializable {

    fun GetX(): Float {
        return X
    }

    fun GetY(): Float {
        return Y
    }
}
